apps/api/src/domain/models/song/commands/create-song.command-handler.ts

```diff
@@ -21,6 +25,19 @@ import { SongCreated } from './song-created.event';
  */
 @CommandHandler(CreateSong)
 export class CreateSongCommandHandler extends BaseCommandHandler<Song> {
+    protected repositoryForCommandsTargetAggregate: IRepositoryForAggregate<Song>;
```

Comment:
We could do away with this if we had a `forAggregate` method on the repository provider.